ui/import-href.js

```diff
@@ -93,3 +93,25 @@ export const importHref = function(href, onload, onerror, optAsync) {
   }
   return link;
 };
+
+/**
+ * Convenience method for importing inline HTML.
+ *
+ * This method imports all the elements to the document's body
+ * and inlines all the scripts to allow fot their execution.
+ *
+ * @param {string} html HTML to import.
+ */
+export const importHTML = (html) => {
+  const tmpl = document.createElement('template');
+  tmpl.innerHTML = html;
+  [...tmpl.content.children].forEach((el) => {
+    if (el.tagName === 'SCRIPT' && !el.src) {
+      const script = document.createElement('script');
+      [...el.attributes].forEach((attr) => script.setAttribute(attr.name, attr.value));
+      script.setAttribute('src', `data:text/javascript;charset=utf-8,${encodeURIComponent(el.textContent)}`);
```

Comment:
Sorry, didn't have time to test it, but: does this make it hard to debug the script  on the borwser?

Reply:
without adding some source maps I guess it does, yes but I think the logics in the importHref polyfill do that so we can improve this too.